### ISSUE TITLE
Add the ability to hide the history of certain banners.

### ISSUE THREAD

**Issue by Vaskass** [MadeBaruna/paimon-moe#114]
Add the ability to hide the history of certain banners on the Wish Counter page. For example, hide the Beginners' Wish banner history and the page will look something like this.
![image](https://user-images.githubusercontent.com/46313746/127369071-a2302b8a-978e-40bc-8661-d7c98e3de250.png)

**Comment by Sahasrahla** [comment]
I agree with the ability to hide certain widgets on this page, but would also like the ability to rearrange them.

What I'd like to see is the Character/Weapon/Standard wish counter at the top as they are, but then the second row of widgets be each banner's stats widget in the same order.
